addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/RemoteControllerService.java

```diff
@@ -81,13 +92,55 @@ public void removeEventListener(EventListener listener) {
         listeners.remove(listener);
     }
 
+    public boolean checkConnection() {
+        return remoteController != null;
+    }
+
+    RemoteController remoteController = null;
+
     @Override
     public void start() {
-        // nothing to start
+        if (remoteController != null) {
+            try {
+                remoteController.openConnection();
+            } catch (RemoteControllerException ignore) {
```

Comment:
Is this really an exception you would like to ignore?

Reply:
Good question. Couple of options: if exception then remove the current object and try to create connection afresh; or ignore the exception (but in that case perhaps add a log message). If the remoteController!=null it means that in the past we were successful in using it, so the TV might just be switched off.

I think I will leave it like this, but add a log message. Agree?